web/runtest.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Fixed - per discussion on Skype we'll leave this in, but I'm adding comments so future users will know what the script is about/for.